build.gradle

```diff
@@ -1,143 +1,27 @@
-	// 3. Add the server keys to your known hosts: ssh-keyscan -p 2222 ext1.cqse.eu >> ~/.ssh/known_hosts
-	//   - Simple "ssh ext1.cqse.eu" won't work because you need the rsa version of the server key for the SSH library
-	//     used by the gradle-ssh-plugin to work
-	// 4. (Optional) Add this entry to your ~/.ssh/config to be able to login to CQSE share via ssh on the command line
-	// 	Host ext1.cqse.eu
-	//    	Port 2222
-	//    	HostName ext1.cqse.eu
-	//    	IdentityFile ~/.ssh/id_rsa
-	//   	User www-data
-	// 5. Now the deploy job should work.
-	// 6. If you have issues with invalid private keys convert them to the older format using:
-	//    puttygen <private-key-path> -O private-openssh -o <old-format-private-key-path>
-	cqseMavenRepositoryPrivateKey = System.getProperty("user.home") + '/.ssh/id_rsa'
-	cqseMavenRepositoryHost = 'ext1.cqse.eu'
-	cqseMavenRepositoryPort = 2222
-	cqseMavenRepositoryMavenPath = '/var/www/share/exchange/public/maven'
-	cqseMavenRepositoryUrl = "scp://$cqseMavenRepositoryHost:$cqseMavenRepositoryPort$cqseMavenRepositoryMavenPath"
-	cqseMavenRepositoryUser = 'www-data'
 }
 
 subprojects {
-	apply plugin: 'java'
-	apply plugin: 'org.hidetake.ssh'
-
-	sourceCompatibility = 1.8
-	targetCompatibility = 1.8
-
-	tasks.withType(JavaCompile) {
-		sourceCompatibility = "1.8"
-		targetCompatibility = "1.8"
+	if (VersionUtils.INSTANCE.isTaggedRelease()) {
+		version appVersion
+	} else {
+		version appVersion + '-SNAPSHOT'
 	}
-
-	group 'com.teamscale'
-
-	configurations {
-		deployerJars
-	}
-
-	repositories {
-		maven {
-			url 'https://share.cqse.eu/svn/artifacts/maven'
-		}
-		jcenter()
-	}
-
-	dependencies {
-		deployerJars 'org.apache.maven.wagon:wagon-ssh:2.2'
-	}
-
-	remotes {
-		share {
-			host = cqseMavenRepositoryHost
-			port = cqseMavenRepositoryPort
-			user = cqseMavenRepositoryUser
-			identity = file(cqseMavenRepositoryPrivateKey)
-		}
-	}
-
-	tasks.withType(Upload) {
-		doFirst {
-			if (it.name == 'install') {
-				return
-			}
-
-			ssh.run {
-				session(remotes.share) {
-					// Gradle gets stuck when we do not delete these metadata files before publishing the new version
-					def artifactName = project.name
-					if (artifactName == 'agent') {
-						artifactName = 'teamscale-jacoco-agent'
-					}
-					remove cqseMavenRepositoryMavenPath + '/com/teamscale/' + artifactName + '/maven-metadata.xml'
-					remove cqseMavenRepositoryMavenPath + '/com/teamscale/' + artifactName + '/maven-metadata.xml.md5'
-					remove cqseMavenRepositoryMavenPath + '/com/teamscale/' + artifactName + '/maven-metadata.xml.sha1'
-				}
-			}
-		}
-	}
-
 }
 
-// Installs the plugin to your local maven repository
-task installGradlePlugin(dependsOn: [
-		':agent:install',
-		':impacted-test-engine:install',
-		':report-generator:install',
-		':teamscale-gradle-plugin:install',
-		':teamscale-client:install',
-		':tia-client:install'
-])
+// To install the artifacts locally run publishToMavenLocal
```

Comment:
shouldn't I run `iGP` the special task you set up below?

Reply:
This instruction is more general. `iGP` just installs everything except the agent (because it is not changed often when working on the plugin and takes long to build because of the shadow plugin)